package.json

```diff
@@ -3,10 +3,16 @@
   "scripts": {
     "postinstall": "bower install",
     "build": "pulp build",
-    "test": "pulp test"
+    "test": "pulp test",
+    "spago-build": "spago build",
+    "spago-test": "spago -x ./test.dhall test"
   },
   "devDependencies": {
-    "pulp": "^12.3.0",
-    "purescript": "^0.12.0"
+    "pulp": "^15.0.0",
+    "purescript": "^0.14.0",
+    "big-integer": "^1.6.48"
+  },
+  "dependencies": {
+    "big-integer": "^1.6.48"
```

Comment:
This shouldn't be a proper dependency, it's enough that it's in the `devDependencies`.